src/main/scala/org/pantsbuild/jarjar/JJProcessor.scala

```diff
@@ -0,0 +1,18 @@
+package org.pantsbuild.jarjar
```

Comment:
Could we move this into  `package sbtassembly`? If Pants is using that name, I don't want to collide with it.